modules/navier_stokes/src/fvbcs/INSFVOutletPressureBC.C

```diff
@@ -9,22 +9,50 @@
 
 #include "INSFVOutletPressureBC.h"
 #include "INSFVPressureVariable.h"
+#include "Function.h"
 
 registerMooseObject("NavierStokesApp", INSFVOutletPressureBC);
 
 InputParameters
 INSFVOutletPressureBC::validParams()
 {
-  InputParameters params = FVFunctionDirichletBC::validParams();
+  InputParameters params = FVDirichletBCBase::validParams();
   params += INSFVFullyDevelopedFlowBC::validParams();
+
+  // Value may be specified by a AD functor (typically a variable), a function or a postprocessor
+  params.addParam<FunctionName>("function", "The boundary pressure as a regular function");
```

Comment:
Should we make this deprecated? Hmm I see that this could be important since the function returns Reals as opposed to the ADReal functor ... ?

Reply:
yeah that's why I want to keep it.